packages/components/src/components/Spacer/Spacer.test.tsx

```diff
@@ -8,12 +8,12 @@ describe('Spacer', () => {
 
```

Comment:
Both `default size` test names (lines 6 and 24) need to be updated to:
`(12)`
or remove the size from the test